### PR TITLE
Impact Summit 2020 is done

### PR DESCRIPTION
**Pull request by nikhilro**

### CODE REVIEW

File: summit/index.html [hunk 1]
```diff
@@ -588,8 +524,7 @@ <h4>What can I unreasonably expect?</h4>
 						<h4>Who can attend?</h4>
 						<p>
 							Anyone who thinks critically about the challenges we’re facing in the 21st century and has the guts to do
-							something about them! You can sign up <a target="_blank"
-								href="https://airtable.com/shrYtyHTCeHsdZszH">here</a>.
+							something about them!.
```
Comment (Holben888) [inline summit/index.html:527]: Messed up the punctuation! Change "!." to "!"

File: summit/index.html [hunk 2]
```diff
@@ -601,12 +536,12 @@ <h4>Who are some of the past speakers you’ve invited to The Impact Summit?</h4
 								target="_blank" href="https://drive.google.com/open?id=1yiO-HcIWlUsRiD4YFFXf23OU_DJE5Nx6">Closing
 								Day</a> for a complete list of all past speakers.
 						</p>
-						<br />
+						<!-- <br />
```
Comment (Holben888) [inline summit/index.html:539]: Might as well just delete these, since this section exists in the git history 🤷

File: summit/index.html [hunk 3]
```diff
@@ -486,8 +422,8 @@ <h2>The Social Startup Competition</h2>
 						Working on a startup or non-profit that’s addressing the UN Sustainable Development Goals? Apply to pitch at
 						the summit and share your work with us!
 					</p>
-					<a class="btn btn-warning mt-3" role="button" aria-pressed="true" target="_blank"
-						href="https://airtable.com/shr6UXPWQZEWdhbHL">Apply here</a>
+					<!-- <a class="btn btn-warning mt-3" role="button" aria-pressed="true" target="_blank"
```
Comment (Holben888) [inline summit/index.html:425]: Just delete 🤷

**Comment by nikhilro** [comment]
@Holben888 thanks for the suggestions, made the changes! my original intent was to preserve some stuff I anticipated we'd need again pretty soon and save a split second but you're right, not worth setting undesirable precedents.

**Pull request merged**